controllers/serverTableController.js

```diff
@@ -1,27 +1,65 @@
-var app = angular.module('pokemonDraft', []);
-app.controller('serverTableController', function($scope, $http) {
+var app = angular.module('pokemonDraft', ['ui.bootstrap.modal', 'ui.bootstrap.tpls']);
+app.controller('serverTableController', function($scope, $http, $modal) {
+	// server list
+	$scope.hostedGames = [];
+	$scope.usersID;
+
+    $scope.openModal = function(item) {
+        var modalInstance = $modal.open({
+            templateUrl: 'views/joinGame.html',
+            backdrop: 'static',
+            controller: function($scope, $modalInstance, $sce, item) {
+            	$scope.clone = item;
+                $scope.close = function() {
+                    $modalInstance.dismiss('cancel');
+                };
+            }
+        });
+    }
 
 
     var socket = io.connect('http://localhost:4200');
     socket.on('connect', function(data) {
-        socket.emit('join', 'Hello World from client');
+    	// default namespace/room (its a lobby)
+        socket.emit('initialConnection', 'New client has joined :)');
+
+        // Gets users socket ID
+        $scope.usersID = socket.io.engine.id
     });
 
-    
 
-	// Testing array :)
-	$scope.hostedGames = [
-	{name: "Greg", hostName: "George", players: "1/4", private: "No"}, 
-	{name: "Cool Game", hostName: "Jim", players: "1/2", private: "Yes"},
-	{name: "Fun Game", hostName: "Tim", players: "1/3", private: "No"},
-	{name: "Crazy Game", hostName: "Old'greg", players: "1/5", private: "Yes"},
-	{name: "Interesting Game", hostName: "Jay", players: "1/4", private: "Yes"}
-	];
+	$scope.joinGame = function(id){
```

Comment:
If possible install eslint on sublime, it's a module which will let u know if u disobey a style guide. In airbnb's style guide it is recommended to have a space after a function decleration eg

`$scope.joinGame = function(id) {`